googlesource/source.go

```diff
@@ -32,32 +32,22 @@ type Source struct {
 	sdk.UnimplementedSource
 	bqReadClient *bigquery.Client
 	sourceConfig googlebigquery.SourceConfig
-	// if a user specified table IDs in the config, then the tables here will be those same table IDs.
-	//  However, if the table IDs in config are not specified, then the tables here will contain the list of all tables
-	tables []string
-	//  for all the function running in goroutine we needed the ctx value. To provide the current
+	// for all the function running in goroutine we needed the ctx value. To provide the current
 	// ctx value ctx was required in struct.
-	ctx            context.Context
-	records        chan sdk.Record
-	positions      positions
+	ctx     context.Context
+	records chan sdk.Record
+	// position faces race condition. So will always use it inside lock
```

Comment:
It would be good to clarify what's that race condition. In future, we may be able to fix it.